x/concentrated-liquidity/swaps.go

```diff
@@ -86,13 +98,13 @@ var (
 // two has no liquidity and can be skipped.
 func (ss *SwapState) updateSpreadRewardGrowthGlobal(spreadRewardChargeTotal sdk.Dec) {
 	ss.globalSpreadRewardGrowth = ss.globalSpreadRewardGrowth.Add(spreadRewardChargeTotal)
```

Comment:
The comment above the function says that if liquidity is zero this function is a no-op, but this like gets set regardless. Not sure if it's the code or the comment that's correct.  @ValarDragon you introduced this in https://github.com/osmosis-labs/osmosis/pull/5537, do you know?

Reply:
I think we can move this line to be below the liquidity check for clarity.

This is still a no-op because if liquidity is zero for some reason, out math would compute zero `spreadRewardChargeTotal` so this addition will have no effect